### DISCUSSION TITLE
Is there any way to integrate excalidraw with angular application?

### DISCUSSION THREAD

**Discussion by Shilpaspatil** [excalidraw/excalidraw#3922] [Q&A] [answered]
Is there any way to integrate excalidraw with angular application?

**Comment by dwelle** [comment, marked as answer]
Since Excalidraw depends on React, you'll need to add React into your application in some way. If you don't want to install it via `npm` and add it to your bundle, you can include it via a `<script>` tag.

See https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#in-browser
